src/wasm-io.h

```diff
@@ -29,7 +29,7 @@ namespace wasm {
 
 class ModuleIOBase {
 protected:
-  bool debugInfo;
+  bool debugInfo = true;
```

Comment:
This default should never have been used, if it wasn't set before. Why add one?

If we need one, how about `false` which is safer? (missing debug info is obvious; extra debug info can subtly bloat)

Reply:
It looked dangerous to not have a default, but I can remove this. I don't think it's necessary for anything else in the PR.